巴哈姆特/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "author": {
+      "name": "蒟蒻",
+      "id": "276337222196658178"
+    },
+    "service": "巴哈姆特",
+    "description": {
+      "en": "The largest animation and game community website for Chinese, providing ACG daily news, popular rankings, and a wealth of discussion and communication space, as well as brilliant personal audio and video live and blog posts.",
+      "zh-TW": "華人最大動漫及遊戲社群網站，提供ACG 每日最新新聞、熱門排行榜，以及豐富的討論交流空間，還有精采的個人影音實況、部落格文章。"
+    },
+    "url": "forum.gamer.com.tw",
+    "version": "1.0.0",
+    "logo": "https://i2.bahamut.com.tw/baha_logo5.png",
+    "thumbnail": "https://www.gamer.com.tw/favicon.ico",
```

Comment:
thumbnail must be a screenshot of the website

Reply:
or use a image that is related to the website.

Reply:
Okay, edited.